.github/workflows/docker-tests.yml

```diff
@@ -16,8 +16,6 @@
 name: docker-tests
 
 on:
-  schedule:
-    - cron: "0 2 * * *"
```

Comment:
For consideration (maybe not a blocker) - the docker tests will now only trigger if the docker files themselves or application assemblies are changed.

While this will assure any such changes don't break things, we'll miss the possibility of assuring that changes within other modules haven't broken the assembly and running of the applications, e.g. if a change is made to `nifi.properties` but not catered for in the docker build and that subsequently prevents the application from running

I can't say for certain that such a thing would be a pertinent, and I agree that reducing the number of GitHub Actions is a good thing, I wonder whether a reduction in frequency (e.g. weekly) rather than removal of the cron schedule would be more sensible in order to retain more of the existing QA

Reply:
That's a good question. At this point, the Docker integration test script is a minimal check that the REST API is accessible. I think changing this to a weekly schedule is a good change for now.